doc/release_notes/release_notes_2.5.rst

```diff
@@ -0,0 +1,189 @@
+version 2.5.
+
+ACRN is a flexible, lightweight reference hypervisor that is built with
+real-time and safety-criticality in mind. It is optimized to streamline
+embedded development through an open-source platform. See the
+:ref:`introduction` introduction for more information.  All project ACRN
+source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, tools, and
+documentation. You can either download this source code as a zip or
+tar.gz file (see the `ACRN v2.5 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v2.5>`_) or
+use Git ``clone`` and ``checkout`` commands::
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v2.5
+
+The project's online technical documentation is also tagged to
+correspond with a specific release: generated v2.5 documents can be
+found at https://projectacrn.github.io/2.5/.  Documentation for the
+latest under-development branch is found at
+https://projectacrn.github.io/latest/.
+
+ACRN v2.5 requires Ubuntu 18.04.  Follow the instructions in the
+:ref:`rt_industry_ubuntu_setup` to get started with ACRN.
+
+
+What's New in v2.5
+******************
+
+
+ACRN Configuration and Build
+============================
+
+The following major changes on ACRN configuration and build process have been
+integrated into v2.5:
+
+- Change 1
+- Change 2
+
+For complete instructions to get started with the new build system, refer to
+:ref:`getting-started-building`. For an introduction on the concepts and
+workflow of the new configuration mechanism, refer to
+:ref:`acrn_configuration_tool`.
+
+Upgrading to v2.5 From Previous Releases
+****************************************
+
+We highly recommended that you follow the instructions below to
+upgrade to v2.5 from previous ACRN releases.
+
+Additional Dependencies
+=======================
+
+Python version 3.6 or higher is required to build ACRN v2.5. You can check the version of
+Python you are using by:
+
+.. code-block:: bash
+
+   $ python3 --version
+   Python 3.5.2
+
+Only when the reported version is less than 3.6 (as is the case in the example above) do
+you need an upgrade. The first (and preferred) choice is to install the latest
+Python 3 from the official package repository:
+
+.. code-block:: bash
+
+   $ sudo apt install python3
+   ...
+   $ python --version
+   Python 3.8.8
+
+If this does not get you an appropriate version, you may use the deadsnakes PPA
+(using the instructions below) or build from source yourself.
+
+.. code-block:: bash
+
+   $ sudo add-apt-repository ppa:deadsnakes/ppa
+   $ sudo apt-get update
+   $ sudo apt install python3.9
+   $ python --version
+   Python 3.9.2
+
+In addition, the following new tools and packages are needed to build ACRN v2.5.
+(If you're already using ACRN v2.4 you should already have these tools.):
```

Comment:
A comma would be good after "v2.4". Also, it looks odd to have that colon after a parenthetical that's a whole sentence.